Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -2245,7 +2267,8 @@ script:
       description: A name that appears in the "From:" header for email sent using
         this alias.
       type: String
-  dockerimage: demisto/google-api:1.0.0.24033
+  #dockerimage: demisto/google-api:1.0.0.24033
+  dockerimage: googleapi
```

Comment:
What is this docker image? is it already in dockerfiles repo? the image should also be tagged as well.

Reply:
This is a custom Docker image. I need to add the 'typing' library to the existing demisto/google-api image.

I create the above image locally to test using `/docker_image_create base=demisto/google-api:1.0.0.24033 name=googleapi dependencies=typing` in XSOAR.

I left this value in as it would need to be change to the new Docker image.

Reply:
Basically this is the only command I have seen in which the Google Client API uses typing. All the other commands don't seem to use it. If I use the standard image `demisto/google-api:1.0.0.24033` then I receive errors when running the `gmail-list-labels`. Error is along the lines of `unknown library import 'typing'`.

Reply:
PR is in place for this now -> https://github.com/demisto/etc/issues/47444

Will update the Docker image once it becomes available.